leveler/image_generators.py

```diff
@@ -378,7 +384,9 @@ def make_levelup_image(
         white_text = (250, 250, 250, 255)
         dark_text = (35, 35, 35, 230)
         level_up_text = self._contrast(info_color, white_text, dark_text)
-        lvl_text = "LEVEL {}".format(userinfo["servers"][str(server.id)]["level"])
+        lvl_text = "LEVEL {}".format(
+            self._humanize_number(userinfo["servers"][str(server.id)]["level"])
```

Comment:
Noted, although it might be a good idea to add some sort of function to add human readability. As levels get higher and higher, the number becomes increasingly difficult to read. Consider using the [`redbot.core.utils.chat_formatting.humanize_number`](redbot.core.utils.chat_formatting.humanize_number) method to make the numbers a bit more readable (i.e., 12345 -> 12,345 or 12.345 depending on locale)